setupGlobals.ts

```diff
@@ -0,0 +1,4 @@
+import { TextDecoder as UtilTextDecoder, TextEncoder as UtilTextEncoder } from 'util';
+
+global.TextDecoder = UtilTextDecoder as unknown as typeof globalThis.TextDecoder;
```

Comment:
this is the setup of jest test env 😅  I'm unsuse how to my ts happy with it but since it's for tests didnt feel too scary to me but I'm prolly missing something